<?xml version="1.0"?>
<launch>
    <node pkg="dynamic_reconfigure" type="dynparam" name="$(anon dynparam)" 
        args="load move_base/DWAPlannerROS $(find snap_misc)/config/DWAPlannerROS.yaml"/>
</launch>
